api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityServiceTest.java

```diff
@@ -16,63 +25,140 @@
 import static org.mockito.Mockito.verifyZeroInteractions;
 import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantTestDataFactory.aValidClaimant;
 import static uk.gov.dhsc.htbhf.claimant.testsupport.EligibilityResponseTestDataFactory.anEligibilityResponse;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithVouchers;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithZeroVouchers;
 import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.DUPLICATE;
 import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.ELIGIBLE;
+import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.INELIGIBLE;
 
 @ExtendWith(MockitoExtension.class)
 class EligibilityServiceTest {
 
     @InjectMocks
-    private EligibilityService eligibilityService;
+    EligibilityService eligibilityService;
+
+    @Mock
+    EligibilityClient client;
 
     @Mock
-    private EligibilityClient client;
+    EligibilityStatusCalculator eligibilityStatusCalculator;
 
     @Mock
-    private EligibilityStatusCalculator eligibilityStatusCalculator;
+    ClaimRepository claimRepository;
 
     @Mock
-    private ClaimRepository claimRepository;
+    CycleEntitlementCalculator cycleEntitlementCalculator;
 
     @Test
     void shouldReturnDuplicateResponseWhenLiveClaimAlreadyExists() {
         given(claimRepository.liveClaimExistsForNino(any())).willReturn(true);
         Claimant claimant = aValidClaimant();
 
-        EligibilityResponse eligibilityResponse = eligibilityService.determineEligibilityForNewClaimant(claimant);
+        EligibilityAndEntitlement eligibilityResponse = eligibilityService.determineEligibilityForNewClaimant(claimant);
 
         assertThat(eligibilityResponse.getEligibilityStatus()).isEqualTo(DUPLICATE);
         verify(claimRepository).liveClaimExistsForNino(claimant.getNino());
-        verifyZeroInteractions(client, eligibilityStatusCalculator);
+        verifyZeroInteractions(client, eligibilityStatusCalculator, cycleEntitlementCalculator);
+    }
+
+    @Test
+    void shouldReturnEligibleWhenNoDuplicateAndEligibleWithVouchers() {
```

Comment:
`NotDuplicate`?

Reply:
I was thinking 'when there is no duplicate' - either works